server/channels/app/plugin.go

```diff
@@ -684,6 +685,20 @@ func (a *App) mergePrepackagedPlugins(remoteMarketplacePlugins map[string]*model
 			},
 		}
 
+		if !isEnterpriseLicense {
```

Comment:
Can we add a comment clarifying that this all works only because we don't actually list Playbooks in the marketplace? (Otherwise, the v2 would have shown up before this anyway.)